Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1918,66 +1937,134 @@ void CGContextSetPatternPhase(CGContextRef context, CGSize phase) {
     textTransform.ty = std::round(textTransform.ty);
 
     CGAffineTransform finalTextTransform = CGAffineTransformConcat(textTransform, deviceTransform);
-    if ((fabs(finalTextTransform.a * glyphRun->fontEmSize) <= c_glyphThreshold &&
-         fabs(finalTextTransform.b * glyphRun->fontEmSize) <= c_glyphThreshold) ||
-        (fabs(finalTextTransform.d * glyphRun->fontEmSize) <= c_glyphThreshold &&
-         fabs(finalTextTransform.c * glyphRun->fontEmSize) <= c_glyphThreshold)) {
-        TraceWarning(TAG, L"Glyphs too small to be rendered");
+    for (size_t i = 0; i < runsCount; ++i) {
```

Comment:
I'm not so sure.  This is only a problem on our platform, but idk if we'd just want to draw the rest of a line if part is so small that it would be causing a crash.